extensions/amp-list/0.1/test/test-amp-list-load-more.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpDocService} from '../../../../src/service/ampdoc-impl';
+import {AmpList} from '../amp-list';
+import {Services} from '../../../../src/services';
+import {isExperimentOn, toggleExperiment} from '../../../../src/experiments';
+
+
+describes.realWin('amp-list component', {
+  amp: {
+    ampdoc: 'single',
+    extensions: ['amp-list'],
+  },
+}, env => {
+  let win, doc, ampdoc, sandbox;
+  let element, list;
+  let templates;
+
+  beforeEach(() => {
+    win = env.win;
+    doc = win.document;
+    ampdoc = env.ampdoc;
+    sandbox = env.sandbox;
+
+    templates = {
+      findAndSetHtmlForTemplate: sandbox.stub(),
+      findAndRenderTemplate: sandbox.stub(),
+      findAndRenderTemplateArray: sandbox.stub(),
+    };
+    sandbox.stub(Services, 'templatesFor').returns(templates);
+    sandbox.stub(AmpDocService.prototype, 'getAmpDoc').returns(ampdoc);
+
+    toggleExperiment(win, 'amp-list-load-more', true);
```

Comment:
Can you put this in the test spec? That is:

```javascript
describes.realWin('amp-list component', {
  amp: {
    ampdoc: 'single',
    extensions: ['amp-list'],
    experiments: ['amp-list-load-more'],
  },
}, env => {
```

Seems like you should be able ot.

Reply:
I've tried this and haven't been able to get it to work, which is why I've asserted that the test is on in various places. But I'll look into this a bit more.

Reply:
I haven't quite figured out exactly why it doesn't work, but there's another case of it here: 

https://github.com/ampproject/amphtml/blob/420db9d7187a935585e54dbacca175a82e916109/extensions/amp-auto-lightbox/0.1/test/test-carousel-criteria.js#L25-L48

Here's a usage of `experiments: []` passed to `describes.realWin`. If you take out the `toggleExperiment` and try to assert `isExperimentOn`, it fails. Other usages of this in our codebase only exist in `describes.integration` and `describes.endtoend`.